etc/notebook/jupyter_notebook_config.py

```diff
@@ -0,0 +1,3 @@
+import os
+if 'ROOTSYS' in os.environ:
+    c.NotebookApp.extra_static_paths.append(os.path.join(os.environ['ROOTSYS'], 'js/'))
```

Comment:
Show we add following lines for the case when ROOTSYS is not configured:
```
import os, subprocess
prefix_path = subprocess.run(["root-config", "--prefix"], capture_output=True).stdout.decode().strip()
c.NotebookApp.extra_static_paths = [os.path.join(prefix_path, 'js')]
```
Also I not sure if `c.NotebookApp.extra_static_paths.append` should be used - may be just direct assign.

Reply:
You are right, currently an installation would still be broken. Your idea is great, we could just rely on `root-config`! I'll have a look :)

Reply:
And we'll carry over the other paths, in case of `ROOTSYS` this not needed because `thisroot.sh` will set it anyway.